apps/console/src/features/applications/components/help-panel/oidc-configurations.tsx

```diff
@@ -89,7 +89,7 @@ export const OIDCConfigurations: FunctionComponent<OIDCConfigurationsPropsInterf
     });
 
     return (
-        <>
+        <Form>
```

Comment:
Shall we add a screenshot of the look and feel of the info tabs in OIDC & SAML apps?

Reply:
**OIDC Look and feel**
![image](https://user-images.githubusercontent.com/66235437/136010329-8c58e056-721b-40ad-a387-6a35ca4c040b.png)

**SAML app look and feel**
![image](https://user-images.githubusercontent.com/66235437/136010698-aec11352-7533-4627-91c7-f4674c2473bf.png)